packages/fuels-core/src/lib.rs

```diff
@@ -29,6 +29,31 @@ pub type Selector = ByteArray;
 pub type Bits256 = [u8; 32];
 pub type EnumSelector = (u8, Token, EnumVariants);
 
+#[derive(Debug, Clone, PartialEq, Default)]
+pub struct StringToken {
+    pub data: String,
```

Comment:
I'd make this private and create a `new` fn.

Don't see a reason to allow somebody to extract potentially invalid `data` from this struct.